django/views/csrf.py

```diff
@@ -97,12 +97,11 @@
 """
 
 
-def csrf_failure(request, reason=""):
+def csrf_failure(request, reason="", template_name="403_csrf.html"):
```

Comment:
I thought about this when I typed the name for a second time, but I decided against it because this way, the function signature already makes pretty clear how the function behaves -- which is a behaviour I find very useful.